lib/rgeo/geographic/interface.rb

```diff
@@ -403,30 +377,16 @@ def projected_factory(opts = {})
 
       private
 
-      def proj_4055
-        unless defined?(@proj44055)
-          @proj44055 = CoordSys.supported?(:proj4) && CoordSys::Proj4.create("+proj=longlat +a=6378137 +b=6378137 +towgs84=0,0,0,0,0,0,0 +no_defs")
-        end
-        @proj44055
-      end
-
       def coord_sys_4055
         unless defined?(@coord_sys_4055)
-          @coord_sys_4055 = CoordSys::CS.create_from_wkt('GEOGCS["Popular Visualisation CRS",DATUM["Popular_Visualisation_Datum",SPHEROID["Popular Visualisation Sphere",6378137,0,AUTHORITY["EPSG","7059"]],TOWGS84[0,0,0,0,0,0,0],AUTHORITY["EPSG","6055"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.01745329251994328,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4055"]]')
+          @coord_sys_4055 = CoordSys::CONFIG.default_coord_sys_class.create(4055)
```

Comment:
what is the point of these methods? Faster initialisation ?

Reply:
The `Geographic` module contains some factories that allow you to project from 4326 or 4055 to another coord sys, so these just contain the definitions for the source coord systems of the factories.